x-pack/plugins/enterprise_search/public/applications/shared/doc_links/doc_links.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { docLinksService } from './doc_links';
+
+describe('DocLinksService#setDocLinks()', () => {
+  it('adds links from docLinks', () => {
+    const docLinks = {
+      DOC_LINK_VERSION: '',
+      ELASTIC_WEBSITE_URL: 'https://elastic.co',
+      links: {
+        enterpriseSearch: {
+          base: 'http://elastic.enterprise.search',
+          appSearchBase: 'http://elastic.app.search',
+          workplaceSearchBase: 'http://elastic.workplace.search',
+        },
+      },
+    };
+
+    docLinksService.setDocLinks(docLinks as any);
+
+    expect(docLinksService.enterpriseSearchBase).toEqual(docLinks.links.enterpriseSearch.base);
+    expect(docLinksService.appSearchBase).toEqual(docLinks.links.enterpriseSearch.appSearchBase);
+    expect(docLinksService.workplaceSearchBase).toEqual(
+      docLinks.links.enterpriseSearch.workplaceSearchBase
+    );
+    expect(docLinksService.cloudBase).toEqual(
+      `${docLinks.ELASTIC_WEBSITE_URL}guide/en/cloud/current`
+    );
```

Comment:
This is definitely my personal preference but I'd really like it if we just repeated the static strings instead of referencing them. Performance isn't a question or issue when it comes to tests; developer understanding is paramount, and it's way easier for me to understand what "final output" looks like and connect it to the original input when they're static

```suggestion
    expect(docLinksService.enterpriseSearchBase).toEqual('http://elastic.enterprise.search');
    expect(docLinksService.appSearchBase).toEqual('http://elastic.app.search');
    expect(docLinksService.workplaceSearchBase).toEqual('http://elastic.workplace.search');
    expect(docLinksService.cloudBase).toEqual(`https://elastic.coguide/en/cloud/current`);
```

The cloud link looks a little odd btw, I might fix up the mock URL?